sys/include/net/loramac.h

```diff
@@ -102,6 +102,13 @@ extern "C" {
  * @name    LoRaMAC default values
  * @{
  */
+/**
+ * @brief Default LoRaWAN region
+ */
+#if !defined(LORAMAC_REGION_EU868) && !defined(LORAMAC_REGION_IN865)
+#define LORAMAC_REGION_EU_868                  (1)
```

Comment:
```suggestion
#define LORAMAC_REGION_EU_868                  1
```

Reply:
Sorry, to jump-in late, but I think there define name should rather be `LORAMAC_REGION_EU868` (no `_` between EU and 868). This notation is used everywhere in the lorawan specs.

I haven't checked in detail what the rationale between the addition of this `_`, maybe there's a good reason ?

Reply:
Dash helps with readability to identify the frequency `EU_868`, at least for me. Alliance do not use '_' though.

Reply:
I think I would semtech pkg notation here, since we already configure the pkg with that suffix